main.cpp

```diff
@@ -13,75 +13,495 @@ What you create in this project will be used as the basis of Project 5 in the co
+    //detail the car interior
+    void detailInterior( Car car );
+    
+    //5) a member variable whose type is a UDT.
+    Car carBeingServiced;  
+};
+
+
+/*
+Thing 1) Can plant
+5 properties:
+    1) tin amount
+    2) raw fish amount
+    3) autoclave capacity
+    4) cans per day
+    5) worker/workhours
+3 things it can do:
+    1) produce canned fish
+    2) load boxes with goods in warehouse
+    3) produce bio scrap
+ */
+
+struct canPlant
+{
+    //tin amount
+    int tinAmount = 1000;
+    //raw fish amount
+    double rawFish = 100.21;
+    //autoclave capacity
+    int autoclaveLoad = 300;
+    //cans per day
+    int dayCansOut = 900;
+    //worker/workhours
+    float workHours = 12;
+    
+    //Can plant office worker
+    struct OfficeWorker
+    {
+        //operator Id
+        int callOperatorId = 1;
+        //internal phone number
+        int intNumber;
+        //name
+        std::string operatorName = "Elena";
+        //working hours start
+        float hourStart = 5.5f;
+        //working hours end
+        float hourEnd = 12.0f;
         
-Sub Object 2: Instrument Cluster
-    Name 4 things you'll find on the:   Instrument Cluster
-        1)
-        2)
-        3)
-        4)
-    Name 3 things you can do with the:   Instrument Cluster
-        1)
-        2)
-        3)
+        //make a call
+        void makeCall(int customerId=0, float time=0);
+        //pickup a call
+        void pickCall(int number, float time);
+        //reserve the order
+        int reserveGood(int customerId, int amount=0);
+    };
+    
+    //produce canned fish
+    int produceCans(double slicedFish, int tinAmount);
+    //load boxes with goods in warehouse
+    void loadBoxes(int boxes, int wareHouseNumber);
+    //produce bio scrap
+    double reportScrapOut(double rawFish, float workHours);
+};
     
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
-    Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1)
-        2)
-        3)
-    Name 2 things you can do with the:   Seat
-        1)
-        2)
-*/
+/*
+Thing 2) SIP provider
+5 properties:
+    1) name
+    2) number of simultaneous calls
+    3) FAX support protocol
+    4) price
+    5) SLA conditions
+3 things it can do:
+    1) do calls
+    2) charge customer
+    3) lease traffic
+ */
+struct sipProvider
+{
+    //supported codecs
+    std::string codecName = "G729";
+    //number of simultaneous calls
+    int calls=2;
+    //FAX support protocol
+    bool t38 = false;
+    //price
+    double price=0;
```

Comment:
be consistent with your formatting.  
check your spacing around the `=`
- [ ] revise